framer/Material-UI.framerfx/package.json

```
@@ -15,9 +15,9 @@
     "url": "https://github.com/mui/material-ui/issues"
   },
   "dependencies": {
-    "@mui/icons-material": "^5.3.1",
-    "@mui/material": "^5.4.0",
-    "@mui/utils": "^5.3.0",
+    "@mui/icons-material": "^5.3.2",
+    "@mui/material": "^5.4.1",
+    "@mui/utils": "^5.3.1",
```

To match the latest version? x the other dependencies?

```suggestion
    "@mui/utils": "^5.4.1",
```